modules/nodes-sigstore-configure-image-policy.adoc

```diff
@@ -0,0 +1,203 @@
+----
++
+Scheduling is disabled on each node as it is updated.
+
+.Verification
+
+. After the nodes in your cluster are updated, you can verify that the image policy has been configured:
+
+.. Enter into the node in debug mode by running the following command::
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+.. Set `/host` as the root directory within the debug shell by running the following command:
++
+[source,terminal]
+----
+sh-5.1# chroot /host/
+----
+
+.. Examine the `<namespace>.json` file  by running the following command: 
++
+.Example output
+[source,json]
+----
+sh-5.1# cat /etc/crio/policies/mynamespace.json
+----
++
+.Example output
+[source,json]
+----
+# ...
+  "docker": {
+   "example.com": [
+    {
+     "type": "sigstoreSigned",
+     "keyData": "a2V5RGF0YQ==",
+     "rekorPublicKeyData": "cmVrb3JLZXlEYXRh",
+     "signedIdentity": {
+      "type": "matchRepoDigestOrExact"
+     }
+# ...
+----
+
+////
+..  Check the crio log for sigstore signature verification for namespaced policy by running the following command:
++
+[source,terminal]
+----
+sh-5.1#  journalctl -u crio | grep -A 100 "Pulling image: docker.io"
+----
++
+.Example output
+[source,terminal]
+----
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.704339479Z" level=debug msg="IsRunningImageAllowed for image docker:docker.io/myrepo/rhel8:latest" file="signature/policy_eval.go:274"
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.704371669Z" level=debug msg=" Using transport \"docker\" specific policy section \"docker.io\"" file="signature/policy_eval.go:150"
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.704426760Z" level=debug msg="Reading /var/lib/containers/sigstore/lyman9966/rhel8@sha256=fb51de62cfc8dfdec6a1d7288f75ad18c57ddaf249da7da0552ca8c881b8ee95/signature-1" file="docker/docker_image_src.go:545"
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.704732347Z" level=debug msg="Looking for sigstore attachments in docker.io/myrepo/rhel8:sha256-fb51de62cfc8dfdec6a1d7288f75ad18c57ddaf249da7da0552ca8c881b8ee95.sig" file="docker/docker_client.go:1138"
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.704776952Z" level=debug msg="GET https://registry-1.docker.io/v2/lyman9966/rhel8/manifests/sha256-fb51de62cfc8dfdec6a1d7288f75ad18c57ddaf249da7da0552ca8c881b8ee95.sig" file="docker/docker_client.go:617"
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.882270334Z" level=debug msg="Content-Type from manifest GET is \"application/vnd.oci.image.manifest.v1+json\"" file="docker/docker_client.go:989"
+Dec 06 08:58:37 ip-10-0-94-101 crio[2289]: time="2024-12-06 08:58:37.882464127Z" level=debug msg="Found a sigstore attachment manifest with 1 layers" file="docker/docker_image_src.go:639"
+----
++
+You can see references to sigstore such as: `Looking for sigstore attachments` and `Found a sigstore attachment manifest with 1 layers`.
```

Comment:
https://issues.redhat.com/browse/OCPNODE-2254?focusedId=26282787&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-26282787  This comment clarifies that the log message: `Using transport \"docker\" specific policy section...` indicates the policy.json configuration for Sigstore is being used to verify the signature.